modules/node/src/config/config.service.ts

```diff
@@ -236,24 +236,32 @@ export class ConfigService implements OnModuleInit {
     return this.get(`INDRA_REDIS_URL`);
   }
 
-  async getDefaultPaymentProfile(assetId: string = AddressZero): Promise<PaymentProfile | undefined> {
+  getRebalancingServiceUrl(): string | undefined {
+    return this.get(`INDRA_REBALANCING_SERVICE_URL`);
+  }
```

Comment:
is this env var in the startup scripts anywhere?

should be optional

Reply:
Yeah, that's why it's marked as possibly undefined. This PR does not add the integration of the service yet.